ManatiCR/retopais
# RP-7: Blog Content Type.

https://manati.atlassian.net/browse/RP-7

How to test:
- Go to node/add/blog
- You should have a form with these fields: title, image (inline entity form), body, category (with 3 categories)
- In body field you could embed already created images using last wysiwyg button in first row
